### ISSUE TITLE
Wording : Member ? User ? Something else ? 

### ISSUE THREAD

**Issue by JocelynDelalande** [spiral-project/ihatemoney#302]
I think we can be more consistent in the wording we use to designate the people sharing a budget.

In some places of the code it is "Member" (french: "Membres"), in other "Users" (which we translate by: "Membres" or "Utilisateur" in French). So… We are a little messy on that :-)

To look around at current wording state by yourself, read  [the translation file](https://github.com/spiral-project/ihatemoney/blob/master/ihatemoney/translations/fr/LC_MESSAGES/messages.po).

1. IMHO "user" has no meaning here, there is no link in IHM between people using the tool and people being listed. So I suggest replacing all "user" uses for something else.

2. To go further, I know that in French,  "Participants" would better fit what we want to express than "Membre", as "Membre" suggests formal adhesion/subscription to a group. Ihatemoney have to be neutral on that and not assume that we are speaking about a formal group.

PS: I'm only speaking of changes about end-user strings, no need to rename classes/tables/technical-stuff.

**Comment by almet** [comment]
Hi, thanks for raising this. I agree with you on this, but I think we should also be consistent in our naming in the code. « participant » seems fine in french, and probably is also in english (not sure).

**Comment by almet** [comment]
We should rework the code as well, to be members everywhere, and reflect this in the strings.

**Comment by zorun** [comment]
Currently (10c35e41ce7) we have the following distribution:

| Word | Occurrences in source translation file | Occurrences in code¹ |
|--|--|--|
| people | 3 | 2 |
| member | 3 | 297 |
| participant | 3 | 0 |
| person | 1 | 98 |

Regarding strings, I don't think a single word would be appropriate everywhere, for instance "People to notify" in the email invite form sounds right because they are not yet participants/members.  That being said, it's still a bit of a mess :/

¹For counting code occurrences, I used the snippet below to look at all python files except migrations:
```
for word in people member participant person
do
  find . -path ./ihatemoney/migrations -prune -o -name '*.py' -exec grep -H -i "$word" {} \; | wc -l
done
```

**Comment by Glandos** [comment]
Since #575 goes for "participants", and I agree with this, let's go for this renaming.
Of course "participants" will be used only for people already in the project. When referring to invitation or administration, it still can be "people" or even "user".

**Comment by JocelynDelalande** [comment]
:+1: @glandos @Natim @zorun

That is such a joy when some people craft a solution for an issue you reported a few years back :hugs:

**Comment by zorun** [comment]
@JocelynDelalande sometimes it takes time for good ideas to materialize ;)

@Glandos thanks for mentioning "user", I have updated the statistics table to account for it.

The result is interesting: "user", "participant", "people", "member" are used in the strings, while "member" and "person" are used in the code.  The only overlap is "member"...
Also, updating strings should be easy (14 occurrences), but normalizing the code represents quite some amount of work (~200 occurrences).

**Comment by Glandos** [comment]
That works for me. I would go by first renaming strings, but *maybe* that's because I'm lazy on refactoring.

**Comment by Glandos** [comment]
Closed in #871
